docs/recipes/routing.md

```diff
@@ -18,7 +18,7 @@ class ProtectedPage extends Component {
     authExists: PropTypes.bool,
   }
 
-  componentWillReceiveProps({ authExists }) {
+  componentWillReceiveProps({ !authExists }) {
```

Comment:
This is destructuring off of props - we would want to move this down to line 22 instead of here